performance_curves/performance_curve.py

```diff
@@ -1,35 +1,134 @@
+import operator
 import numpy as np
 import matplotlib.pyplot as plt
 from typing import Callable, Optional
 
 from performance_curves.utils import synchronize_sort
 
 
-class PerformanceCurve:
+class PerformanceCurveLike:
     """Stores values of the x- and y-axis of the performance curve and allows plotting"""
-    def __init__(self, cutoff, metric_value):
-        self.cutoff = cutoff
+    def __init__(self, metric_value, example_counts):
```

Comment:
Great call! 

For array variables, I have changed them all to plural (e.g. `performance_values` or `case_counts`) since the array contains multiple items. 

For consistent terminology, I now use the terms as follows:
- metric, performance, etc --> performance
- observation, case, example, etc. --> case
- score, prediction, probability estimate --> score
- cutoff, threshold, etc --> threshold

I have updated these changes in this PR, but let me know if you think differently.